en/concepts/scene/skybox.md

```diff
@@ -86,40 +93,37 @@ A cube map can be generated manually in Creator from six normal maps by followin
     > 1. The property boxes in CubeMap that do not have a texture asset yet will be populated using the default asset.
     > 2. The six property boxes in CubeMap **do not use the same texture**, otherwise they will not be displayed properly for some platforms.
 
-## Using IBL
+## Diffuse Illumination
 
-If the **UseIBL** property is checked, objects in the scene use specular reflections of the ambient lighting generated by the skybox, while allowing for more accurate diffuse lighting (convolution maps).
+Creator supports the following three types of ambient diffuse lighting, which can be select in the pull-down menu of **Env Lighting Type**.
 
-If the **UseIBL** property is disabled, objects in the scene will only use diffuse lighting in the hemispherical light mode.
+![type](skybox/sky-box-type.png)
 
-### Diffuse Lighting
+The types and descriptions are as follows:
 
-Creator supports the following two types of ambient diffuse lighting, which can be toggled by checking or unchecking the **ApplyDiffuseMap** property, noting that this property is only displayed when **UseIBL** is checked.
-
-1. **Hemispheric Light**: when the **ApplyDiffuseMap** property is unchecked, hemispheric light diffusion will be used. This method is controlled by the **SkyLightingColor** and **GroundLightingColor** properties in the **Ambient** component, and has higher rendering performance, but less detail and poor lighting directionality. **Manually adjustable, but may become inconsistent with the environment map**.
-
-    ![ambient-diffuse](skybox/ambient-diffuse.png)
+| Type | Description |
+| :---| :--- |
+| **HEMISPHERE_DIFFUSE** | hemisphere diffuse |
+| **AUTOGEN_HEMISPHERE_DIFFUSE_WITH_REFLECTION** | hemisphere diffuse and Environment reflection |
+| **DIFFUSEMAP_WITH_REFLECTION** | diffuse convolution map and environment reflection |
 
-2. **Convolution Map**: when the **ApplyDiffuseMap** property is checked, the convolution map diffuse will be used. This method is advanced diffuse reflection, which can correctly express the diffuse lighting generated by environment map, with better lighting directionality and details. However, the convolution map in the **DiffuseMap** property is automatically generated for diffuse reflection and does not allow manual editing.
+1. **Hemisphere Diffuse**: when the **Env Lighting Type** options is **HEMISPHERE_DIFFUSE**, hemispheric light diffusion will be used. This method is controlled by the **SkyLightingColor** and **GroundLightingColor** properties in the **Ambient** component, and has higher rendering performance, but less detail and poor lighting directionality. **Manually adjustable, but may become inconsistent with the environment map**.
```

Comment:
**The properties are manually adjustable, but may become inconsistent with the environment map**